roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,20 +15,27 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
     docker_image_base = {
         "origin": {
             "repo": "openshift",
             "image": "origin",
+            "has_registry_console": False,
```

Comment:
This seems to stretch a lot what `docker_image_base` holds... can we even describe what this structure is?

Both methods `containerized_docker_images` and `rpm_docker_images` return a list with a single item based on the values in `docker_image_base`. Wouldn't it be simpler and clearer to just store a flat list of images for each case (origin / ocp)?